### PR TITLE
lagrange: Add version 1.2.3

### PR DESCRIPTION
**Pull request by earnestma**

### CODE REVIEW

File: bucket/lagrange.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "description": "A Beautiful Gemini Client",
+    "homepage": "https://gmi.skyjake.fi/lagrange",
+    "license": "BSD-2-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/skyjake/lagrange/releases/download/v1.2.2/lagrange_v1.2.2_windows-x64_portable.zip",
+            "hash": "6e86c6b4691167ae31824fc043d7175a984dae572cf2bfa4d140bd4b7cce8379"
+        }
+    },
+    "bin": "lagrange.exe",
```
Comment (Ash258) [inline bucket/lagrange.json:12]: There are no command line parameters available from quick look and the command line is stuck until the window is presented

image


```suggestion
```

File: bucket/lagrange.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "persist": "userdata",
```
Comment (Ash258) [inline bucket/lagrange.json:13]:
```suggestion
```

File: bucket/lagrange.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "shortcuts": [
+        [
+            "lagrange.exe",
+            "Lagrange"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/lagrange.json:19]:
```suggestion
    ],
    "persist": "userdata",
```

File: bucket/lagrange.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.2.2",
+    "description": "A Beautiful Gemini Client",
```
Comment (Ash258) [inline bucket/lagrange.json:3]:
```suggestion
    "description": "Desktop GUI client for browsing Geminispace",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/lagrange.json (hunk 1 above)
Comment (earnestma) [reply bucket/lagrange.json:12]: image

**Review by Ash258 — APPROVED** [review]

**Pull request merged**